forc/src/cli/commands/test.rs

```diff
@@ -90,5 +90,14 @@ pub(crate) fn exec(command: Command) -> Result<()> {
     out.lines().for_each(|line| println!("{}", line.unwrap()));
     thread.join().unwrap();
 
-    Ok(())
+    let child_success = match child.try_wait() {
+        Ok(Some(returned_status)) => returned_status.success(),
+        Ok(None) => child.wait().unwrap().success(),
+        Err(_) => false,
```

Comment:
I would want to see what the error was, should it happen. Maybe a `forc_util::println_red_err` here?

Reply:
The output of `cargo test` is still printed out for me locally. Should be: https://github.com/FuelLabs/sway/blob/65820ef909fb5d225d0ec59a94eea8a28a26ca2c/forc/src/cli/commands/test.rs#L90